scripts/run_benches_for_runtime.sh

```diff
@@ -34,6 +35,8 @@ pallets=(
 	pallet-xcm
 	parachain-staking
 	public-credentials
+	pallet-deposit-storage
```

Comment:
I think these should only be added for Peregrine, not Spiritnet, for now.

Reply:
The runtime can be set as an argument to this script. For spiritnet the command would fail, since the dip pallets are not in the runtime yet. 

I would add them since they will be part of 1.13, and I am afraid to forget it.

Reply:
It's just that few lines below more pallets are added if the runtime is peregrine. So we should stay consistent. Either we remove those lines, or we add the only-Peregrine pallets there.

Reply:
Ah sorry. Its late: [717121b](https://github.com/KILTprotocol/kilt-node/pull/596/commits/717121bddfefcc2870449371aa7eb73c0917f07c)